packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -364,37 +364,24 @@ async function onDidStartDebugSessionHandler(event: vscode.DebugSession): Promis
       (debugConfig.url || debugConfig.port) && // it's from launch.json
       !debugConfig.postRestartTask
     ) {
+      allRunningDebugSessions.add(event.id);
+
       // and not a restart one
       // send f5 event telemetry
-      const localAppId = (await getLocalTeamsAppId()) as string;
-      const isLocal =
-        (debugConfig.url as string) &&
-        localAppId &&
-        (debugConfig.url as string).includes(localAppId);
-      let appId = "";
       let env = "";
-      if (isLocal) {
-        appId = localAppId;
-      } else {
-        if (debugConfig.teamsfxAppId) {
-          appId = debugConfig.teamsfxAppId;
-        }
-        if (debugConfig.teamsfxEnv) {
-          env = getHashedEnv(event.configuration.env);
-        }
+      if (debugConfig.teamsfxEnv) {
+        env = getHashedEnv(debugConfig.teamsfxEnv);
       }
 
       ExtTelemetry.sendTelemetryEvent(TelemetryEvent.DebugStart, {
         [TelemetryProperty.DebugSessionId]: event.id,
         [TelemetryProperty.DebugType]: debugConfig.type,
         [TelemetryProperty.DebugRequest]: debugConfig.request,
         [TelemetryProperty.DebugPort]: debugConfig.port + "",
-        [TelemetryProperty.DebugRemote]: isLocal ? "false" : "true",
-        [TelemetryProperty.DebugAppId]: appId,
+        [TelemetryProperty.DebugRemote]: debugConfig.teamsfxIsRemote + "",
```

Comment:
Is this property needed? Can we determine whether it is local by checking env == sha256("local")?

Reply:
This property exists before multi env feature. Additionally, "local" env is introduced recently in config unify feature. So I think better to keep this property currently and may consider retire it afterwards.

Reply:
It makes sense.